### ISSUE TITLE
Navigation among the views

### ISSUE THREAD

**Issue by Wameedh** [imougy/Planet-Earth-Bugs#11]
<h2>Steps:</h2>
1. Open the app.
2. Go to the Watch List tab.
3. Tap on any watch list
   4.Tap on any stock from your list.
4. Got to the Advanced Chart by Swiping right.
5. Swipe left or right on the Advanced Chart view.

<h2>Results:</h2>
- When you tap on any stock and go to the group of views that are related to it ( Advanced Chart, News, Tweets, Events, Technicals, Competitors, Dividends, Earnings, and Numbers) there is nothing in the UI to inform the user about these views and how to navigate among them. This is very important specially for a first time user.

![evernote camera roll 20160116 155445](https://cloud.githubusercontent.com/assets/13262511/12375206/9b9ffed8-bc6c-11e5-8671-fa6574ebd6d4.png)

![evernote camera roll 20160116 155332](https://cloud.githubusercontent.com/assets/13262511/12375209/b9b88d0e-bc6c-11e5-8d44-768e08600c93.png)
- If you are on the Advanced Chart view you won't be able to swipe left smoothly. Also, if you swipe right you will be guided to the Watch List views 

![evernote camera roll 20160116 155513](https://cloud.githubusercontent.com/assets/13262511/12375210/c46464e4-bc6c-11e5-8b21-026e51e64419.png)

![evernote camera roll 20160116 155731](https://cloud.githubusercontent.com/assets/13262511/12375212/cebd34ac-bc6c-11e5-87cd-0967bb432a4e.png)

<h2>Expectation:</h2>

- Having  an icon or maybe arrows that would inform the user about the selection of the views that are related to the stock would be great. Like the iamge below maybe!

![evernote camera roll 20160116 155619](https://cloud.githubusercontent.com/assets/13262511/12375213/e726bec8-bc6c-11e5-82c4-dc31fe570972.png)
- It should be easy to navigate when the user is on the Advanced Chart view by swiping life and right. Also, when the user swipe right on the Advanced Chart view He/She should be looped around the group of views that are related to the stock that the user is viewing, rather than jump back to the the watch list view that the stock belong to.

Thank you
Wameedh :smile:

**Comment by imougy** [comment]
You are right on both issues. The indicator at the top doesn't tell how many views we have, also the side navigation doesn't work perfectly in the Advanced Chart view.
Thanks,
